clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/UbamTests.scala

```diff
@@ -360,8 +360,8 @@ trait UbamTests { self: BaseIntegrationSpec =>
     }
 
     val upsertData = TransferUbamV1Metadata(
-      sampleAlias = metadata.sampleAlias,
-      project = metadata.project
+      project = metadata.project,
```

Comment:
Not sure what happened here and below on line 655 (and in `TransferUbamV1Metadata.markDeleted` below). Are these intentional changes?

Reply:
No, I probably did a keyboard shortcut in IntelliJ accidentally.